include/nbl/video/alloc/SubAllocatedDescriptorSet.h

```diff
@@ -47,6 +51,28 @@ class SubAllocatedDescriptorSet : public core::IReferenceCounted
 			m_composed->m_totalDeferredFrees -= getWorstCaseCount();
 		}
 
+		DeferredFreeFunctor(const DeferredFreeFunctor& other) = delete;
+		DeferredFreeFunctor& operator=(const DeferredFreeFunctor& other) = delete;
+		inline DeferredFreeFunctor& operator=(DeferredFreeFunctor&& other)
+		{
+			m_composed = other.m_composed;
+			m_addresses = other.m_addresses;
+			m_binding = other.m_binding;
```

Comment:
you need to add the nullification of `other` here, don't ask why, its just weird that way

Reply:
please don't mark things as resolved yourself, makes me loose track, adn you still don't set `other.m_ = nullptr`